prepare-commit-msg

```diff
@@ -15,32 +27,12 @@ add_jira_issue_id_prefix() {
 }
 
 add_co_authored_by_trailers() {
-  co_authored_by_trailers=$(git mob --list | sed "s/^/Co-authored-by: /")
-  trailer_args=""
-
-  OLDIFS="$IFS"
-  IFS='
-'
-  for line in $co_authored_by_trailers; do
-    trailer_args="${trailer_args} --trailer \"$line\""
-  done
-  IFS="$OLDIFS"
-
-  if [ -n "$trailer_args" ]; then
-    eval "git interpret-trailers --if-exists addIfDifferent ${trailer_args} --in-place $1"
-    printf "%s\n\n" "$co_authored_by_trailers"
-  fi
-}
-
-main() {
-  # Do nothing during rebase
-  [ -z "$(git branch --show-current)" ] && exit 0
-
-  # Do nothing during amend (without message) or reuse of a commit message
-  [ "$2" = 'commit' ] && exit 0
-
-  add_jira_issue_id_prefix "$@"
-  add_co_authored_by_trailers "$@"
+  git mob --list |
+    sed "s/^/Co-authored-by: /" |
+    tee /dev/tty |
+    sed "s/^/--trailer\n/" |
+    tr '\n' '\0' |
+    xargs -0 git interpret-trailers --if-exists addIfDifferent --in-place "$1"
```

Comment:
Currently a newline is printed after the Co-authored-by trailers are printed. Could you please add the following:

```suggestion
    xargs -0 git interpret-trailers --if-exists addIfDifferent --in-place "$1"
  printf "\n" 
```

![image](https://github.com/Mubashwer/git-mob/assets/4198792/7ebd298e-beaa-4f2c-b183-1ad2fca12ec9)

Reply:
Welp I went ahead and opened another PR with a new test! Perhaps we can merge that first.

Reply:
The integration test is now failing: `stderr="tee: /dev/tty: No such device or address\n"`

Reply:
Okay I think it's ready now. I stopped using /dev/tty, that seems kinda iffy.

Thanks so much for reviewing and accepting changes!